demisto_sdk/commands/common/errors.py

```diff
@@ -1825,6 +1830,10 @@ def file_cannot_be_deleted(file_path: str):
     def suggest_fix(file_path: str, *args: Any, cmd: str = "format") -> str:
         return f'To fix the problem, try running `demisto-sdk {cmd} -i {file_path} {" ".join(args)}`'
 
+    @staticmethod
+    def suggest_fix_missing_unittest(file_path: str) -> str:
+        return f"To fix the problem, add an unittest file for {file_path}"
+
```

Comment:
this is only called once, let's use an f-string there instead of defining this method and calling it there.